File: 02_fizz_buzz/fizzbuzz.py
def fizzbuzz():
  result_array = []

for number in range(1, 101):
  if number % 15 == 0:
  result_array.append("fizzbuzz")
elif number % 3 == 0:
  result_array.append("fizz")
elif number % 5 == 0:
  result_array.append("buzz")
else :
  result_array.append(number)

return result_array

print(fizzbuzz())

File: 02_fizz_buzz/fizzbuzz.js
const fizzBuzz = () => {
    resultArray = [];
    for (let i = 1; i <= 100; i++) 
        if (i % 3 === 0 && i % 5 === 0) {
            resultArray.push("fizzbuzz");
        }
    else if (i % 3 === 0) {
        resultArray.push("fizz");
    } else if (i % 5 === 0) {
        resultArray.push("buzz");
    } else {
        resultArray.push(i);
    }
    return resultArray;
};

console.log(fizzBuzz());
